ffi/diplomat/js/examples/tinywasm/build.sh

```diff
@@ -21,18 +21,22 @@ BASEDIR=$(dirname "$(realpath "$0")")
 
 # Build the WASM library
 # TODO: This likely doesn't work if $BASEDIR has spaces
-RUSTFLAGS="-Cpanic=abort -Copt-level=s -C link-arg=-zstack-size=${WASM_STACK_SIZE} -Clinker-plugin-lto -Ccodegen-units=1 -C linker=${BASEDIR}/ld.py -C linker-flavor=wasm-ld" cargo +${ICU4X_NIGHTLY_TOOLCHAIN} build \
+RUSTFLAGS="-Cpanic=abort -Copt-level=s -C link-arg=-zstack-size=${WASM_STACK_SIZE} -Clinker-plugin-lto -Ccodegen-units=1 -C linker=${BASEDIR}/ld.py -C linker-flavor=wasm-ld" \
+# Special linker flags shouldn't overwrite the top-level target directory
+CARGO_TARGET_DIR="target" \
+cargo +${ICU4X_NIGHTLY_TOOLCHAIN} build \
     -Z build-std=std,panic_abort -Z build-std-features=panic_immediate_abort \
     --target wasm32-unknown-unknown \
     --release \
-    --package icu_capi_tinywasm
+    --package icu_capi_cdylib \
```

Comment:
An advantage of tinywasm, and something I was trying to demonstrate, was how you could build a WASM file out-of-tree with your own settings. We already have multiple examples of how to build WASM in-tree.

Reply:
The comment in the root `Cargo.toml` said it was because you didn't want to mess up the root `target`, which can be solved differently. Non-workspace crates are really annoying, for example ecma402 doesn't build anymore and was even forgotten during the 1.0 release. If you insist that tinywasm should also demonstrate out-of-tree building on top of the actual linking tinyfication, I'll revert, but I don't think it's great.

Reply:
Also the out-of-workspace fuzzer crates are broken.